### PR TITLE
Tools

### PR DESCRIPTION
**Pull request by samsklar11**

### CODE REVIEW

File: product/conversational-ai/tools.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+title: "Tools"
+description: "Provide your agent with real time information and the ability to take action in third party apps with external function calls."
+---
+
+Tools allow you to make external function calls to third party apps so you can get real-time information. You might use tools to:
+- Schedule appointments on someone's calendar
+- Book restaurant reservations
+- Create or update customer records in a Customer
```
Comment (louisjoecodes) [inline product/conversational-ai/tools.mdx:9]:
```suggestion
- Create or update customer records in a CRM system
```

File: product/conversational-ai/tools.mdx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+Method: Get
+
+URL: https://api.cal.com/v2/slots/available
+
+(2) Once our caller and receptionist have agreed on a suitable time to meet, we'd like to book a meeting. For this we use the [Create a booking](https://cal.com/docs/api-reference/v2/bookings/create-a-booking#create-a-booking) endpoint. 
+
+Name: Book_Meeting
+
+Description: Call this function once you have collect the caller's full name, meeting time, and email address.
```
Comment (louisjoecodes) [inline product/conversational-ai/tools.mdx:42]:
```suggestion
Description: Call this function once you have collected the caller's full name, meeting time, and email address.
```

File: product/conversational-ai/tools.mdx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+In our Description for each, we define a prompt that our Conversational Agent will use to extract the relevant information from the call transcript using an LLM.
+
+Here's how we defined our body parameters for our AI receptionist:
+| Identifier  | Data Type | Required | Description                                                                                                                                                                                      |
+|-------------|-----------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| startTime   | String    | Yes      | The start time of the slot the person is checking availability for in UTC timezone, formatted as ISO 8601 (e.g., '2024-08-13T09:00:00Z'). Extract time from natural language and convert to UTC. |
+| eventTypeId | String    | Yes      | The desired meeting length, as an event id. If 15 minutes, return 1351800. If 30 minutes, return 1351801. If 60 minutes, return 1351802. If you are not sure, return 1351801.                    |
```
Comment (louisjoecodes) [inline product/conversational-ai/tools.mdx:98]:
```suggestion
| eventTypeId | String    | Yes      | The unique Cal event ID for the meeting duration. Use 1351800 for a 15-minute meeting, 1351801 for 30 minutes, and 1351802 for 60 minutes. If no specific duration is provided, default to 1351801 (30 minutes).                   |
```

**Review by louisjoecodes — CHANGES_REQUESTED** [review]

**Comment by samsklar11** [comment]
Thanks @louisjoecodes - committed those changes

**Review by louisjoecodes — APPROVED** [review]

**Pull request merged**